notebooks/MAGICC7.ipynb

```diff
@@ -225,56 +430,46 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "running RCP26\n"
+      "running rcp26\n"
      ]
     },
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_CH4_CONC.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_CH4_TAUOH.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_CH4_TAUTOT.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_CH4N_EMIS.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_CH4T_EMIS.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_N2O_CONC.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_N2O_TAUSCALE.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_N2O_TAUTOT.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_N2ON_EMIS.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_N2OT_EMIS.BINOUT'>\n",
-      "unclosed file <_io.BufferedReader name='../../magicc/out/DAT_SURFACE_TEMP.BINOUT'>\n"
+      "/Users/seangtkelley/miniconda3/envs/climate-model-pymagicc/lib/python3.10/site-packages/pymagicc/io/scen7.py:43: UserWarning: MAGICC6 RCP region naming (R5*) is not compatible with MAGICC7, automatically renaming to MAGICC7 compatible regions (R5.2*)\n",
+      "  warnings.warn(warn_msg)\n",
+      "/Users/seangtkelley/miniconda3/envs/climate-model-pymagicc/lib/python3.10/site-packages/pymagicc/io/scen7.py:43: UserWarning: MAGICC6 RCP region naming (R5*) is not compatible with MAGICC7, automatically renaming to MAGICC7 compatible regions (R5.2*)\n",
+      "  warnings.warn(warn_msg)\n"
      ]
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "running RCP45\n",
-      "running RCP60\n",
-      "running RCP85\n"
+     "ename": "ValueError",
+     "evalue": "You have more than one `FILE_EMISSCEN_X` flag set. Using more than one emissions scenario is hard to debug and unnecessary with Pymagicc's Dataframe scenario input. Please combine all your scenarios into one Dataframe with Pymagicc and Pandas, then feed this single Dataframe into Pymagicc's run API.",
```

Comment:
I cannot get passed this error using a .CFG file identical to the one above. I've also tried removing references to `FILE_EMISSCEN_X` and I get the same error. Any ideas?